File: Python_Server/bp/Cemetery.py
import Card


class Cemetery:
    def __init__(self):
        """
        墓地
        """
        self.card_list = []

    # 墓地にカードを追加する
    def add(self, card: Card):
        """
        カードを墓地に置く
        :param card: 墓地に置くカード
        """
        self.card_list.append(card)

    # 墓地のカードを取る
    def get(self, length: int) ->Card:
        """
        墓地のカードを選択する
        :param length: 何番目のカードか
        """
        return self.card_list.pop(length)

File: server/core/Cemetery.js

var Cemetery = class{
	constructor(){
		this.cardList = [];
	}

	add(card){
		this.cardList.push(card);
	}
	
	get(length){
		var card = this.cardList[length];
		card.splice(length, 1);
		return card;
	}

};

module.exports = Cemetery;

/*
var Cemetery = function() {
	this.cardList = [];
};

Cemetery.prototype.add = function(card){
	this.cardList.push(card);
};

Cemetery.prototype.get = function(length){
	var card = this.cardList[length];
	card.splice(length, 1);
	return card;
};


module.exports = Cemetery;
 */